code/Model/DynamoDbSession.php

```diff
@@ -44,26 +44,32 @@ public function getHandler()
     public static function get()
     {
         // Use DynamoDB for distributed session storage if it's configured
-        if (defined('AWS_DYNAMODB_SESSION_TABLE') && AWS_DYNAMODB_SESSION_TABLE) {
-            $dynamoOptions = array('region' => AWS_REGION_NAME);
+        $AWS_DYNAMODB_SESSION_TABLE = getenv('AWS_DYNAMODB_SESSION_TABLE');
+        if (!empty($AWS_DYNAMODB_SESSION_TABLE)) {
+            $AWS_REGION_NAME = getenv('AWS_REGION_NAME');
```

Comment:
Can you please clean up the variable names? The standard is `$awsRegionName` and such :)